typescript error in jumplinks

https://github.com/patternfly/patternfly-elements/runs/7568139954?check_suite_focus=true#step:6:1010

#2073 revealed a typescript error in https://github.com/patternfly/patternfly-elements/blob/cd12cef723427a87e46524614582195a001ad00f/elements/pfe-jump-links/pfe-jump-links-nav.ts#L1029-L1033

> error TS2774: This condition will always return true since this function is always defined. Did you mean to call it instead?

--- a/.changeset/warm-fishes-listen.md
+++ b/.changeset/warm-fishes-listen.md
@@ -1,0 +1,5 @@
+---
+"@patternfly/pfe-jump-links": patch
+---
+
+Fix predicate which updates light DOM

--- a/elements/pfe-jump-links/pfe-jump-links-nav.ts
+++ b/elements/pfe-jump-links/pfe-jump-links-nav.ts
@@ -172,7 +172,7 @@ export class PfeJumpLinksNav extends LitElement {
   private scrollTimeout?: number;
 
   /** This flag indicates if the rebuild should update the light DOM */
-  private updateLightDom = false;
+  private shouldUpdateLightDOM = false;
 
   private currentActive?: number;
 
@@ -797,7 +797,7 @@ Alternatively, add the \`autobuild\` attribute to dynamically generate the list 
       return;
     }
 
-    this.updateLightDom = true;
+    this.shouldUpdateLightDOM = true;
     this.rebuild();
   }
 
@@ -810,7 +810,7 @@ Alternatively, add the \`autobuild\` attribute to dynamically generate the list 
       return;
     }
 
-    this.updateLightDom = true;
+    this.shouldUpdateLightDOM = true;
 
     // Reset the sections object to allow refetching
     this._sections = null;
@@ -1026,7 +1026,7 @@ Alternatively, add the \`autobuild\` attribute to dynamically generate the list 
 
       let menu;
 
-      if (this.autobuild && this.update) {
+      if (this.autobuild && this.shouldUpdateLightDOM) {
         menu = this.build();
       } else {
         menu = this.querySelector('ul');
@@ -1050,7 +1050,7 @@ Alternatively, add the \`autobuild\` attribute to dynamically generate the list 
       });
     }
 
-    this.updateLightDom = false;
+    this.shouldUpdateLightDOM = false;
   }
 
   /**

